src/js/itemset.js

```diff
@@ -274,6 +282,12 @@ export default {
         } );
         input.value = value;
         label.appendChild( input );
+        /**
+         * #510 Show warning if select_multiple value has spaces
+         */
+        if ( ( attributes[ 'data-type-xml' ] == 'select' ) && ( attributes[ 'type' ] == 'checkbox' ) && ( value.indexOf( ' ' ) > -1 ) ) {
+            dialog.alert( t( 'selectpicker.valuehasspaces.multiple', { value: value } ) );
+        }
```

Comment:
I think it may be better to move your check to before `createOption` or `createInput` are called. Do you agree?

Reply:
sure, update code position

Reply:
The remaining issue is that it doesn't for select_multiple with a minimal appearance. I tested it by removing the other 2 `<select>` (without minimal appearance) from the XForm.

Reply:
update the condition to include select_multiple with a minimal appearance.

Reply:
Thanks @theywa. It works but now it also catches a single select with minimal (try by changing `<select>` to `<select1>` in the xform and it also catches autocomplete select_one questions.

I think this would work well instead:

```js
const multiple = ( inputAttributes[ 'data-type-xml' ] == 'select' && inputAttributes[ 'type' ] == 'checkbox' ) || ( $list[ 0 ] && $list[ 0 ].multiple );
```

Reply:
@MartijnR - Yes, you're right. I miss checking minimal for select1, will be careful next time. I just update the code and how about tests? is it okay like that?